test/correctness/conv3x3a16.cpp

```diff
@@ -0,0 +1,123 @@
+#include "Halide.h"
+#include <iostream>
+
+//#define RDOM_OK 1
+
+using namespace Halide;
+using namespace Halide::Internal;
+
+int main(int arch, char **argv) {
+    int verbose = 0;
```

Comment:
If we keep this (see comment below on the usage of this), this probably should be const bool.